src/pages/main-page/main-page-pure.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC, memo } from 'react';
+
+import SC from './styled';
+import localization from '../../lib/localization';
+import {
+  SearchBox,
+  SearchBoxTitle,
+  SearchLinks
+} from '../../components/search-box/search-box';
+import HeaderSC from './components/header/styled';
+import NewsList from './components/news-list/news-list-component';
+import SearchEntities from '../../components/search-entities/search-entities.component';
+
+interface Props {
+  recentEntities?: any;
+  losItems?: any;
+  newsItems?: any;
+}
+
+const MainPage: FC<Props> = ({
+  recentEntities = [],
+  losItems = {},
+  newsItems = []
+}) => (
+  <div>
+    <SearchBox>
+      <SearchBoxTitle large>{localization.searchBoxHeader}</SearchBoxTitle>
+      <SearchLinks />
+    </SearchBox>
+    <main id="content" className="container">
+      <SC.Content className="row">
+        <section className="col-12 col-lg-8">
+          <HeaderSC.Header>
+            <span>{localization.sort.modified}</span>
+          </HeaderSC.Header>
+          <SearchEntities entities={recentEntities} losItems={losItems} />
+        </section>
+        <section className="col-12 col-lg-4">
+          <NewsList newsItems={newsItems} />
+        </section>
+      </SC.Content>
+    </main>
+  </div>
+);
+
+export default memo(MainPage);
```

Comment:
Kan bruke "withReferenceData" HOC for å hente nødvendig los. Se implementasjon på detaljsiden

Reply:
Gjort